Cql/Cql.Packaging/CqlCrosswalk.cs

```diff
@@ -247,7 +247,7 @@ public CqlCrosswalk(TypeResolver typeResolver)
         }
     }
 
-    internal class TypeEntry
+    public class TypeEntry

```

Comment:
This feels like a record. We should remove the setters for the properties.  Maybe we can even make this a `record` class?